offset_sample_count: 5  # number of camera strobe/image pairs to collect for offset calculation
offset_sample_time: 0.5  # [s] time to wait after requesting a camera image

# camera
camera_rate: 20  # [Hz] camera frame rate commanded by teensy

# imu
gyro_sens: 1  # [0,3] gyro full-scale range and sensitivity
acc_sens: 1  # [0,3] accel full-scale range and sensitivity
imu_filter_size: 5  # window size for low-pass filter on imu

# input topics
image_topics: [/camera/depth/image_rect_raw,
               /camera/infra1/image_rect_raw,
               /camera/infra2/image_rect_raw]
info_topics: [/camera/depth/camera_info,
              /camera/infra1/camera_info,
              /camera/infra2/camera_info]
metadata_topics: [/camera/depth/metadata,
                  /camera/infra1/metadata,
                  /camera/infra2/metadata]
